src/main/resources/application-test.yml

```diff
@@ -0,0 +1,27 @@
+server:
+  port: 8080
+
+jwt:
+  secret: oneweek
```

Comment:
maybe it is not good idea to push secret code of jwt😅

Reply:
This value is specified in the application-test.yml file, so it is not an actual value